app/views/rent_mailer/new_rent_email.slim

```diff
@@ -1,12 +1,15 @@
 
 h1 
-  | You have registered a new rent for the book 
+  = t('rent_mailer.new_rent.body.title') 
+  '
```

Comment:
Can I have an explication on this?

Reply:
Yes, that's to add a space between the translated text and the book name

Reply:
I tried '<=' and '>='  to add the space at the end of those lines but that didn't work so I decided to add the space in the translation text.

Reply:
Perfect 👍